### PR TITLE
Linking my blog to the website

### PR DESCRIPTION
**Pull request by hshafaq7**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -251,4 +251,10 @@
   title: Interviewing Contributors !!
   img: Kothari.jpg
   author_name: Yash Kothari
+  
+- url:https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title:Here I come Coding In
+  img:https://user-images.githubusercontent.com/34957702/34479624-e24cb40c-efcd-11e7-83f4-7921fe396574.JPG
+  author_name:Haider Shafaq
+
```
Comment (nikit19) [inline _data/blogs.yml]: Please follow the exact format as given in the readme.md 
there should be a space after `:`

**Review by nikit19 — CHANGES_REQUESTED** [review]
You need to upload the image

File: _data/blogs.yml [hunk 2]
```diff
@@ -251,4 +251,10 @@
   title: Interviewing Contributors !!
   img: Kothari.jpg
   author_name: Yash Kothari
+  
+- url: https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title: Here I come Coding In
+  img: https://user-images.githubusercontent.com/34957702/34479624-e24cb40c-efcd-11e7-83f4-7921fe396574.JPG
```
Comment (abishekvashok) [inline _data/blogs.yml:257]: Dont paste the image url here. Put the image name after uploading to img/blog/

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: _data/blogs.yml (hunk 2 above)
Comment (hshafaq7) [reply _data/blogs.yml:257]: But uploads are diasabled

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Clone the `repo`, then locally do the changes and then push and send a PR.

**Review by ParthS007 — COMMENTED** [review]
@hshafaq7 Please go through `README.md` on how to add a blog and make a new PR.You can ask on FOSSASIA gitter channel if you face any difficulty. 👍

**Pull request closed without merging**